source/NeutronaWand/NeutronaWand.ino

```diff
@@ -5285,6 +5236,59 @@ void wandSerialSend(int i_message) {
   wandComs.sendDatum(sendStruct);
 }
 
+// Helper method to play a sound effect using certain defaults.
+void playEffect(int i_track_id, bool b_track_loop = false, int8_t i_track_volume = i_volume, bool b_fade_in = false, unsigned int i_fade_time = 0) {
+  if(i_track_volume < i_volume_abs_min) {
+    i_track_volume = i_volume_abs_min;
+  }
+  if(i_track_volume > 10) {
+    i_track_volume = i_volume_abs_max;
+  }
+
+  if(b_fade_in == true) {
+    w_trig.trackGain(i_track_id, i_volume_abs_min);
+    w_trig.trackPlayPoly(i_track_id, true);
+    w_trig.trackFade(i_track_id, i_track_volume, i_fade_time, 0);
+  }
+  else {
+    w_trig.trackGain(i_track_id, i_track_volume);
+    w_trig.trackPlayPoly(i_track_id, true);
+  }
+
+  if(b_track_loop == true) {
+    w_trig.trackLoop(i_track_id, 1);
+  }
+}
+
+void stopEffect(int i_track_id) {
```

Comment:
I tried looking at passing an array of tracks to stop, but the code was getting too complex and Arduino doesn't seem to like creating an array on the fly due to strong typing. This is still better than just the direct WavTrigger calls throughout the code (and abstracts the audio player away from the code)